tests/legacy-cli/e2e/utils/project.ts

```diff
@@ -212,22 +214,22 @@ export function useCIChrome(projectDir: string) {
         chromeOptions: {
           args: [
             "--enable-logging",
-            "--no-sandbox",
-            ${process.env['TRAVIS'] ? '"--headless", "--disable-gpu"' : ''}
+            // "--no-sandbox",
```

Comment:
Looks like we can drop this comment, and the one below, now.

Reply:
Left them there for now in case appveyor is brought back in.